hooks/command

```diff
@@ -63,6 +63,37 @@ downloader() {
   fi
 }
 
+check_latest_version() {
+  local _executable="monorepo-diff-buildkite-plugin"
+  local _repo="https://api.github.com/repos/buildkite-plugins/monorepo-diff-buildkite-plugin"
+  local _version=""
+
+  if check_cmd curl; then
+    _version=$(curl -s "${_repo}/releases/latest" | grep -oE '"tag_name": "v[0-9]+\.[0-9]+\.[0-9]+"' | cut -d'"' -f4)
+  elif check_cmd wget; then
+    _version=$(wget -qO- "${_repo}/releases/latest" | grep -oE '"tag_name": "v[0-9]+\.[0-9]+\.[0-9]+"' | cut -d'"' -f4)
+  fi
+
+  if [[ "$_version" =~ ^v[0-9]+\.[0-9]+\.[0-9]+ ]]; then
+    true
+  else
+    _version=""
+  fi
+
+  RETVAL="$_version"
+}
+
+check_binary_version() {
+  local _executable_version_file="monorepo-diff-buildkite-plugin.version"
```

Comment:
given that this value must be synchronized among 2 different functions it would make sense to make it a global variable instead